examples/api/lib/widgets/widget_state/widget_state_mouse_cursor.0.dart

```diff
@@ -0,0 +1,65 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/material.dart';
+
+/// Flutter code sample for [WidgetStateMouseCursor].
+
+void main() {
+  runApp(const WidgetStateMouseCursorExampleApp());
+}
+
+class WidgetStateMouseCursorExampleApp extends StatelessWidget {
+  const WidgetStateMouseCursorExampleApp({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    return MaterialApp(
+      home: Scaffold(
+        appBar: AppBar(title: const Text('WidgetStateMouseCursor Sample')),
+        body: const Center(
+          child: WidgetStateMouseCursorExample(
+            // TRY THIS: Switch to get a different mouse cursor while hovering ListTile.
+            enabled: false,
+          ),
```

Comment:
I'm wondering: rather than recommending that the user tweak the program, what if we have a button or switch below the ListTile that can toggle the `enabled` value?

Not a strong preference though.